### PR TITLE
[RLlib contrib] add contributing.md to RLlib contrib

### PR DESCRIPTION
**Pull request by avnishn**
Add a contributing.md to the rllib contrib directory.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by gjoliver — APPROVED** [review]

File: rllib_contrib/contributing.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Contributing Guidelines
+
+Any issues that are filed in `rllib_contrib` will be solved best-effort by the community and there is no expectation of maintenance by the RLlib team.
```
Comment (kouroshHakha) [inline rllib_contrib/contributing.md:3]: should we introduce a special tag for better book keeping?

File: rllib_contrib/contributing.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+We will not accept contributions that generally add a significant maintenance burden. In this case users should instead make their own repo with their contribution, **using the same guidelines as this repo**, and the RLlib team can help to market/promote it in the ray docs.
+
+## Contributing new algorithms
+
+If users would like to contribute a new algorithm tor rllib_contrib, they should follow these steps:
+1. Create a new directory with the same structure as the other algorithms.
+2. Add a `README.md` file that describes the algorithm and its usecases.
+3. Create unit tests/shorter learning tests and long learning tests for the algorithm.
+4. Submit a PR and a RLlib maintainer will review it and help you set up your testing to integrate with the CI of this repo.
```
Comment (kouroshHakha) [inline rllib_contrib/contributing.md:23]: Submit a PR and tag with rllib_contrib

**Review by kouroshHakha — APPROVED** [review]

File: rllib_contrib/contributing.md (hunk 2 above)
Comment (avnishn) [inline rllib_contrib/contributing.md:23]:
```suggestion
4. Submit a PR and tag with rllib_contrib and a RLlib maintainer will review it and help you set up your testing to integrate with the CI of this repo.
```

Comment (avnishn) [inline rllib_contrib/contributing.md:23]:
```suggestion
4. Submit a PR, add the tag `rllib_contrib`, and then a RLlib maintainer will review it and help you set up your testing to integrate with the CI of this repo.
```

Comment (avnishn) [reply rllib_contrib/contributing.md]: done

**Review by kouroshHakha — APPROVED** [review]

**Review by kouroshHakha — APPROVED** [review]

**Pull request merged**